Documentation/operations/upgrade.rst

```diff
@@ -40,8 +40,7 @@ file.
       helm template |CHART_RELEASE| \\
         --namespace=kube-system \\
         --set preflight.enabled=true \\
-        --set agent.enabled=false \\
-        --set config.enabled=false \\
+        --set agent=false \\
```

Comment:
@seanmwinn what about the `config=false`?

Reply:
we discussed this on yesterday's tech talk and the consensus was this is leftover from pre-helm and nobody actually uses it in the current implementation